docs_dev/releasing.md

```diff
@@ -179,27 +186,51 @@ Alternatively you can use the UI.
    ```
    * Set the tag to be the correct version for the tag.
 
-## Version the website
+# Version the website
 
 The main Kubeflow website at [www.kubeflow.org](www.kubeflow.org) points to the
-**master** branch of the `kubeflow/website` repo. Similarly, 
-[master.kubeflow.org](https://master.kubeflow.org/) also points to the master
-branch.
+version of the website we want users to see.
+
+* Most of the time this will correspond to the **master** branch of the `kubeflow/website` repo
+
+* However, leading up to the release of vY.0.0 [www.kubeflow.org](www.kubeflow.org) 
```

Comment:
Based on my comment below about what I think "vY.0.0" means, I think it'd be clearer to add the word "**major**" here. When I first read this section, I thought you were talking about a patch release, and was therefore rather confused.

Something like this:

```
However, leading up to the next **major** release, vY.0.0, [www.kubeflow.org](www.kubeflow.org) will point at ...
```

Reply:
PTAL